tbapy/main.py

```diff
@@ -1,109 +1,343 @@
-import requests
+from requests import get
+from .models import *
```

Comment:
For some reason it does seem to be necessary. Not sure why. But it was giving me an error if I didn't put it in there. On second thought, I don't think I have to put it in `__init__.py`.